recipes/oommf/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+TCLTKVERSION=8.5
+CONDA_ROOT_PATH=$(conda info --root)
+
+# Compile OOMMF
+export OOMMF_TCL_CONFIG=${PREFIX}/lib/tclConfig.sh
+export OOMMF_TK_CONFIG=${PREFIX}/lib/tkConfig.sh
+cd oommf
+tclsh${TCLTKVERSION} oommf.tcl pimake distclean
```

Comment:
```
conda config --add channels conda-forge
conda install oommf
```
Above should work. I think since there are incompatibilities between defaults and conda-forge, you should recommend the above command. `conda install -c conda-forge <pkg>` might not work sometimes. Having said that, using `tclsh8.5` makes sense if you want to keep compatibility with defaults.